pkg/features/kube_features.go

```diff
@@ -858,7 +859,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	PreferNominatedNode:                            {Default: false, PreRelease: featuregate.Alpha},
 	ProbeTerminationGracePeriod:                    {Default: false, PreRelease: featuregate.Alpha},
 	RunAsGroup:                                     {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.22
-	PodDeletionCost:                                {Default: false, PreRelease: featuregate.Alpha},
+	PodDeletionCost:                                {Default: false, PreRelease: featuregate.Beta},
```

Comment:
Beta features should be enabled by default

Reply:
oops

Reply:
done.